src/WebJobs.Script.WebHost/Models/HostAssignmentContext.cs

```diff
@@ -21,6 +21,9 @@ public class HostAssignmentContext
         [JsonProperty("lastModifiedTime")]
         public DateTime LastModifiedTime { get; set; }
 
+        [JsonProperty("MSISpecializationPayload")]
```

Comment:
Assuming you can't rename the json property because it's an established contract at this point with Antares?